frontend/src/lib/components/Skeleton/Skeleton.scss

```diff
@@ -0,0 +1,25 @@
+.Skeleton {
+    // Background is such that at 25% (one full width) the darker fade starts and finishes slightly less than 75% (3 full widths)
+    // With the background size at 400% this gives the impression of movement
+    background: linear-gradient(90deg, var(--border-light) 25%, var(--border) 45%, var(--border-light) 65%);
```

Comment:
This is more of a design question, but could we turn the intensity of these colors down? It doesn't seem like it's necessary for a skeleton to pass strict contrast checks. They're rather suppose to feel ephemeral and just for me our `--border` colors are a bit too strong for this. Compare these two screenshots:

| Previous | New |
| --- | --- |
| <img width="1800" alt="Screen Shot 2022-09-05 at 16 37 49 3" src="https://user-images.githubusercontent.com/4550621/188474491-248d9f5c-cf99-41f0-8b2e-3ef00421c0d5.png">| <img width="1800" alt="Screen Shot 2022-09-05 at 16 38 29 4" src="https://user-images.githubusercontent.com/4550621/188474489-cd6edcd8-1427-4bf0-be57-1ce9c43ea20f.png"> |

Those darker grays make the transition less seamless.

Reply:
One other thing that comes to mind is that _sometimes_ these skeletons appear over non-white background elements... So we might want a darker but semi-transparent color (like the Antd one) to account for these cases

Reply:
@lottiecoxon was owning loading animations and skeleton loaders. These were documented to use the border-light palette option. They're not strictly one color since we're using an animated gradient to indicate activity. I think it's fine to make them lower contrast, but would defer to @lottiecoxon since she was owning this component.

Reply:
After seeing it in action I do agree with Michael on the colours needing to be toned down - hard to judge this without animation - will test more with animations in the future. 

I think with a widened gradient, slowed down and with a lighter palette we will get an outcome that is less jarring. Happy to jump on a call to work through it with you @benjackwhite similar to when we did the spinner.